exporters/inmemory/src/test/java/io/opentelemetry/exporters/inmemory/InMemoryTracingTest.java

```diff
@@ -71,7 +72,15 @@ public void getFinishedSpanItems() {
   @Test
   public void getFinishedSpanItems_sampled() {
     tracing.getTracer().spanBuilder("A").startSpan().end();
-    tracing.getTracer().spanBuilder("B").setSampler(Samplers.alwaysOff()).startSpan().end();
+    TracerSdk tracerSdk = (TracerSdk) tracing.getTracer();
+    TraceConfig originalConfig = tracerSdk.getActiveTraceConfig();
+    tracerSdk.updateActiveTraceConfig(
+        originalConfig.toBuilder().setSampler(Samplers.alwaysOff()).build());
+    try {
+      tracing.getTracer().spanBuilder("B").startSpan().end();
+    } finally {
+      tracerSdk.updateActiveTraceConfig(originalConfig);
```

Comment:
This sort of logic should be in setup/teardown methods (`@Before` and `@After`), rather than cluttering up the test logic with it.

Reply:
Basically I'd agree but here updating the config is only used in this one specific test method, right?

Reply:
Basically I'd agree but here updating the config is only used in this one specific test method, right?

Reply:
Correct and only for that particular Span.